ClientReconnectTest.testReconnectToNewInstanceAtSameAddress

https://hazelcast-l337.ci.cloudbees.com/job/new-lab-fast-pr/8400/

```
java.lang.AssertionError: CountDownLatch failed to complete within 120 seconds, count left: 1
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at com.hazelcast.test.HazelcastTestSupport.assertOpenEventually(HazelcastTestSupport.java:899)
	at com.hazelcast.test.HazelcastTestSupport.assertOpenEventually(HazelcastTestSupport.java:892)
	at com.hazelcast.test.HazelcastTestSupport.assertOpenEventually(HazelcastTestSupport.java:884)
	at com.hazelcast.client.ClientReconnectTest.testReconnectToNewInstanceAtSameAddress(ClientReconnectTest.java:95)
```

--- a/hazelcast-client/src/test/java/com/hazelcast/client/ClientReconnectTest.java
+++ b/hazelcast-client/src/test/java/com/hazelcast/client/ClientReconnectTest.java
@@ -79,7 +79,9 @@ public class ClientReconnectTest extends HazelcastTestSupport {
     public void testReconnectToNewInstanceAtSameAddress() throws InterruptedException {
         HazelcastInstance instance = hazelcastFactory.newHazelcastInstance();
         Address localAddress = instance.getCluster().getLocalMember().getAddress();
-        final HazelcastInstance client = hazelcastFactory.newHazelcastClient();
+        ClientConfig clientConfig = new ClientConfig();
+        clientConfig.getNetworkConfig().setConnectionAttemptLimit(Integer.MAX_VALUE);
+        final HazelcastInstance client = hazelcastFactory.newHazelcastClient(clientConfig);
 
         final CountDownLatch memberRemovedLatch = new CountDownLatch(1);
         client.getCluster().addMembershipListener(new MembershipAdapter() {

